docs/articles/documentation/reference/test-api/requesthook/README.md

```diff
@@ -103,3 +103,35 @@ async onResponse (event) {
     throw new Error('Not implemented');
 }
 ```
+
+{% include notes-warnings/requesthook-onresponse-no-stub-headers.md href='#stub-response-headers' %}
+
+### Stub Response Headers
+
+Before it passes an incoming response to the application, `RequestHook` can change the response headers.
+
+You can use the `RequestHook._onConfigureResponse()` method to substitute response headers, effectively mocking part of the response.
+
+> Important! The `RequestHook._onConfigureResponse()` method is private. Exercise caution when you implement this method in your request hooks.
+
+The example below demonstrates a `RequestHook` that sets a `x-token:token-123` header on the response and removes the `x-frame-options` header from the response.
+
+```js
+class HeadersHook extends RequestHook {
+  constructor (requestFilterRules) {
+    super(requestFilterRules);
+  }
+
+  async onRequest () { }
+  async onResponse () { }
+
+  _onConfigureResponse (event) {
+    super._onConfigureResponse(event);
+
+    event.setHeader('x-token', 'token-123');
+    event.removeHeader('x-frame-options');
+  }
+}
+```
+
+You can not modify the response body with `RequestHook._onConfigureResponse()`.
```

Comment:
You cannot modify the response body. There is no such functionality.

Reply:
That's exactly what it says :D